src/schema.graphql

```diff
@@ -8,6 +8,7 @@ directive @inOrganization on FIELD_DEFINITION
 # The access token passed must be valid in order to proceed.
 directive @isAuthenticated on OBJECT | FIELD_DEFINITION
 
+directive @isOwner on FIELD_DEFINITION
```

Comment:
Please add a comment above the directive explaining what the directive does and where it can be placed.